homework.js

```diff
@@ -0,0 +1,153 @@
+// Create a function summation()
+
+function summation(num) {
+    let result= " " ;
+    let sum =0 ;
+    
+    for (let index = 1; index <= num; index++) {
+
+        result += index + "+" ;
+        sum +=  index ;
+
+    }
+    return console.log("The result is "+result+" = "+sum);
+}
+
+
+//  Create a function summationEven()
+
+function summationEven(num) {
+    let result= " " ;
+    let sum =0 ;
+    
+    for (let index = 1; index <= num; index++) {
```

Comment:
another way to do this is to increase the index by 2 every time

` for (let i = 2; i <= n; i += 2){
        sum +=i;
    }`